### PR TITLE
Update dev

### PR DESCRIPTION
**Pull request by Vinayakj009**
@Inventrom/public-repo-reviewers : Please go through this pull request.

Do remember to go to update dev branch, when looking for the code.
GitHub is NOT LIKE bitbucket, where going to the code takes to to the code being merged.

The master branch allows users to only upload data to the cloud.
Use of API calls to send commands or pull data from the BoltDuino is not possible on the master branch.

Also, when sending data to the cloud, via the current master branch, the users have to specify 6 CSV values on the cloud, regardless of how many values they want to send via the BoltDuino.

This branch solves both these problems, while ensuring that compatibility is maintained for the master branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,15 +25,15 @@ void setup() {
 }
 
 void loop() {
-  boltiot.CheckPoll(digitalRead(2)); //Send data to the bolt cloud, when the bolt polls the arduino,for data.
-/*This function needs to be called reglarly. Calling the CheckPoll function once every seconds is required*/
+  boltiot.processPushDataCommand(digitalRead(2)); //Send data to the bolt cloud, when the bolt polls the arduino,for data.
+/*This function needs to be called reglarly. Calling the processPushDataCommand function once every seconds is required*/
```
Comment (sashoeb94) [inline README.md]: @Vinayakj009 Typo in comment. `once every second`

File: README.md [hunk 2]
```diff
@@ -24,15 +25,15 @@ void setup() {
 }
 
 void loop() {
-  boltiot.CheckPoll(digitalRead(2)); //Send data to the bolt cloud, when the bolt polls the arduino,for data.
-/*This function needs to be called reglarly. Calling the CheckPoll function once every seconds is required*/
+  boltiot.processPushDataCommand(digitalRead(2)); //Send data to the bolt cloud, when the bolt polls the arduino,for data.
```
Comment (sashoeb94) [inline README.md:28]: @Vinayakj009 So the command polls the Arduino for data and then sends it to the cloud?

**Review by sashoeb94 — APPROVED** [review]
Seems good to go.

File: README.md (hunk 2 above)
Comment (Vinayakj009) [reply README.md:28]: The command checks if the Bolt has send a data poll to the Arduino.

File: README.md (hunk 1 above)
Comment (Vinayakj009) [reply README.md]: Correcting it

**Review by Vinayakj009 — COMMENTED** [review]
Corrected the typo in the readme

**Review by sashoeb94 — APPROVED** [review]
Changes are good to go.

**Review by PallaviDutta — APPROVED** [review]

**Review by rsbuilds — APPROVED** [review]
Add the  functions comment for  example files
examples/CommandHandler/CommandHandler.ino

**Pull request merged**